packages/apollo/src/commands/service/list.ts

```diff
@@ -0,0 +1,168 @@
+import { flags } from "@oclif/command";
+import { ProjectCommand } from "../../Command";
+import { GraphQLSchema } from "graphql";
+import sortBy from "lodash.sortby";
+import { table } from "heroku-cli-util";
+import moment from "moment";
+import { ApolloConfig, isServiceProject } from "apollo-language-server";
+import chalk from "chalk";
+import {
+  ListServices_service_implementingServices,
+  ListServices_service_implementingServices_FederatedImplementingServices_services
+} from "apollo-language-server/lib/graphqlTypes";
+
+interface TasksOutput {
+  config: ApolloConfig;
+  implementingServices: ListServices_service_implementingServices | null;
+}
+
+const formatImplementingService = (
+  implementingService: ListServices_service_implementingServices_FederatedImplementingServices_services
+) => {
+  return {
+    name: implementingService.name,
+    url: implementingService.url || "",
+    updatedAt: `${moment(implementingService.updatedAt).format(
+      "D MMMM YYYY"
+    )} (${moment(implementingService.updatedAt).fromNow()})`
+  };
+};
+
+function formatHumanReadable({
+  implementingServices,
+  graphName,
+  frontendUrl
+}: {
+  implementingServices: ListServices_service_implementingServices | null;
+  graphName: string | undefined;
+  frontendUrl: string | undefined;
+}): string {
+  let result = "";
+  if (
+    !implementingServices ||
+    implementingServices.__typename === "NonFederatedImplementingService"
+  ) {
+    result =
+      "\nThis graph is not federated, there are no services composing the graph";
+  } else if (implementingServices.services.length === 0) {
+    result = "\nThere are no services on this federated graph";
+  } else {
+    // Create a sorted list of the services.
+    const sortedImplementingServices = sortBy<
+      ListServices_service_implementingServices_FederatedImplementingServices_services
+    >(implementingServices.services, [service => service.name.toUpperCase()]);
+
+    table(
+      [...sortedImplementingServices.map(formatImplementingService)].filter(
```

Comment:
This is spreading an array into an array, which gives you an array. We can simplify like before 😄 

```js
sortedImplementingServices.map(formatImplementingService).filter(
  ...
)
```